src/stash-debug/src/main.rs

```diff
@@ -535,7 +537,9 @@ impl Usage {
                     aws_privatelink_availability_zones: None,
                     system_parameter_sync_config: None,
                     http_host_name: None,
-                    connection_context: None,
+                    connection_context: ConnectionContext::for_tests(Arc::new(
+                        InMemorySecretsController::new(),
+                    )),
```

Comment:
I think you'll probably need to make this change to the `catalog-debug` tool too.